flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/HDFSDataStream.java

```diff
@@ -54,12 +54,7 @@
   private Path dstPath;
   private String inUseSuffix;
   private String processId;
-  protected HDFSDataStream(){
-    logger.info("Init HDFSDataStream");
-    ApplicationContext appCtx = new ClassPathXmlApplicationContext("spring-dao.xml");
-    AutowireCapableBeanFactory acbFactory = appCtx.getAutowireCapableBeanFactory();
-    acbFactory.autowireBean(this);
-  }
+
   private static HDFSDataStream hdfsDataStream;
   public static HDFSDataStream getHDFSDataStream(){
     if(hdfsDataStream==null)hdfsDataStream=new HDFSDataStream();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00122)